docs/content/en/docs/topics/provision-a-cluster/_index.md

```diff
@@ -14,123 +20,176 @@ description: Steps to provision a Cassandra cluster in Kubernetes
 * A Kubernetes environment
 * K8ssandra installed and running in Kubernetes - see [Quick start]({{< ref "getting-started" >}})
 
-## Steps
-
-### Use helm to get the running configuration
-
-For many basic configuration options, you may change values in the deployed YAML files. For example, you can scale up or scale down, as needed, by updating the YAML via `helm` command `--set` parameters.
+## Create a cluster
 
-Let's check the currently running values. First get the list of the installed K8ssandra chart. In this example, assume the `releaseName` was defined as `k8ssandra` on the `helm install` command.
+Suppose we install K8ssandra as follows:
 
 ```bash
-helm list
+helm install my-k8ssandra k8ssandra/k8ssandra -f k8ssandra-values.yaml
 ```
 
-**Output**:
+Assume that `k8ssandra-values.yaml` has these properties:
 
-```bash
-NAME     	  NAMESPACE	 REVISION   UPDATED                               STATUS  	CHART      APP VERSION
-k8ssandra	  default  	 1          2021-03-04 20:49:32.975090399 +0000   UTC	      deployed	 k8ssandra-1.0.0	              
+```yaml
+cassandra:
+  clusterName: my-k8ssandra
+  datacenters:
+  - name: dc1
+    size: 3
 ```
 
-You can specify the name of the installed cluster's `releaseName` to get the full manifest. 
+The helm instal command will result in the creation of a `CassandraDatacenter` object with the size set to 3. The cass-operator product that's installed by K8ssandra will in turn create the underlying StatefulSet that has 3 Cassandra pods.
 
-`helm get manifest k8ssandra`
+## Add nodes
 
-Helm displays full details of the properties defined in each deployed YAML file. 
+Add nodes by updating the size property of the datacenter. Example values file:
 
-### Scale up the cluster
+```yaml
+cassandra:
+  clusterName: my-k8ssandra
+  datacenters:
+  - name: dc1
+    size: 4
+```
 
-To scale up, focus on the `size` property. Let's find the current value:
+Apply the changes with `helm upgrade`:
 
 ```bash
-helm get manifest k8ssandra | grep size
+helm upgrade my-k8ssandra k8ssandra/k8ssandra -f k8ssandra-values.yaml
 ```
 
-**Output**:
+{{% alert title="Tip" color="success" %}}
+Another way to upgrade your K8ssandra cluster is by passing in a `--set` parameter. Example:
 
-```yaml
-.
-.
-.
-    size: 1
-      initial_heap_size: 1G
-      max_heap_size: 1G
-      heap_size_young_generation: 1G
+```bash
+helm upgrade my-k8ssandra k8ssandra/k8ssandra --set cassandra.datacenters\[0\].size=4,cassandra.datacenters\[0\].name=dc1
```

Comment:
I would include the ` --reuse-values` option as well in this example. It tells Helm to reuse the previous values used and anything specified with `--set` will override previous values. Without `--reuse-values` it is easy to make a mistake if you have other, additional properties you previously set.

Reply:
Inc'd your comment with subsequent commit.